saplings/circuits/src/components/Chips.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2018-2020 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Node } from '../data/nodeRegistry';
+
+import './Chips.scss';
+
+export function Chips({ nodes, localNodeID, removeFn }) {
+  return (
+    <div className="chips">
+      {nodes.map(node => {
+        const isLocal = node.identity === localNodeID;
+        return (
+          <Chip
+            node={node}
+            isLocal={isLocal}
+            removeFn={() => removeFn(node)}
+            deleteable={!isLocal}
+          />
+        );
+      })}
+    </div>
+  );
+}
+
+function Chip({ node, isLocal, removeFn, deleteable }) {
+  if (!node) {
+    return '';
+  }
+  return (
+    <div className="chip-group">
```

Comment:
I'm not quite sure how to interpret this structure, is the chip-group surrounding an individual chip?

Reply:
my bad, in refactoring things chip-group ended up becoming obsolete and I forgot to remote it.